contracts/lib/CircularBufferUint.sol

```diff
@@ -47,6 +47,11 @@ contract CircularBufferUint {
      *
      * @param _maxItems Defines how many items this buffer stores before
      *                  overwriting older items.
+     *
+     * \pre Setup can only be done once.
```

Comment:
\pre The function can be called only once.

Reply:
Done

Reply:
Fixed